allauth/account/signals.py

```diff
@@ -1,9 +1,22 @@
-from django.dispatch import Signal
+from django.contrib.auth.signals import user_logged_out as dj_user_logged_out
+from django.dispatch import Signal, receiver
 
 
 user_logged_in = Signal(providing_args=["request", "user"])
 user_logged_out = Signal(providing_args=["request", "user"])
 
+
+@receiver(dj_user_logged_out, dispatch_uid="forwards_user_logged_out")
+def forwards_user_logged_out(sender, **kwargs):
```

Comment:
I can't really think of a use case where it is really required for the signal not to fire. Suppose there is another third party app that hooks up the django logout directly -- given that the user is actually being logged out as would be the case when logging out via allauth, I do think it makes more sense to always fire the signal. Also, for sake of simplicitly of the code, simply relying on the Django signal is a one liner, versus quite some intricate machinery introduced in this PR.